decide/src/main/java/org/example/Decide.java

```diff
@@ -18,7 +21,8 @@ public boolean decideHelper() {
     }
 
     public boolean condition0() {
-        return true;
+        return IntStream.range(0, settings.NUMPOINTS - 1).anyMatch(
+                (index) -> settings.POINTS[index].distance(settings.POINTS[index + 1]) > settings.PARAMETERS.LENGTH1);
```

Comment:
Should the range not go from 0 to (NUMPOINTS - 2) since we compare index to index + 1?